docs/use-the-network/run-a-network-server/buy-an-oui/buy-an-oui.mdx

```diff
@@ -0,0 +1,124 @@
+---
+id: buy-an-oui
+hide_title: true
+sidebar_label: Buy an OUI
+slug: /use-the-network/buy-an-oui
+---
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+# What is an OUI?
+
+Each LoRaWAN Network Server (LNS) on the Helium Network acquires an
+Organizationally Unique Identifier (OUI). This registers the LNS with the
+blockchain and allocates two very important routing objects owned and maintained
+by the OUI operator:
+
+- up to five **filters**, describing the sets (AppEUI, DevEUI) expected by the
+  LNS
+- one or more **slabs** of NwkAddrs, reserving an address space for the OUI's
```

Comment:
Should this be dev address?

Reply:
I think they are pretty interchangeable, but I do think calling it a "DevAddr" slab might be easier since it's the higher level identifier. I'll correct the [LoRaWAN section](https://docs.helium.com/lorawan-on-helium#data-frames-and-nwkaddr-slabs) to be consistent about "slab" indications.